analytics/views.py

```diff
@@ -1160,6 +1160,9 @@ def support(request: HttpRequest) -> HttpResponse:
             elif billing_method == "charge_automatically":
                 update_billing_method_of_current_plan(realm, charge_automatically=True)
                 context["message"] = f"Billing method of {realm.string_id} updated to charge automatically."
+            event_time = timezone_now()
+            RealmAuditLog.objects.create(realm=realm, acting_user=request.user, event_type=RealmAuditLog.REALM_BILLING_METHOD_CHANGED,
```

Comment:
This should go inside `update_billing_method_of_current_plan` similar to other comments. Log the old and new billing methods as well in `extra_data`